### PR TITLE
s390x: Add test to check if ipl device is set correctly

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

File: tests/installation/bootloader_s390.pm [hunk 1]
```diff
@@ -262,9 +262,11 @@ sub run() {
 
     # we also want to test the formatting during the installation if the variable is set
     if (!get_var("FORMAT_DASD_YAST") && !get_var('S390_DISK') && !get_var('UPGRADE')) {
-        format_dasd;
+        #    format_dasd;
     }
 
+
+
```
Comment (okurz) [inline tests/installation/bootloader_s390.pm]: wut?

File: tests/installation/install_and_reboot.pm [hunk 2]
```diff
@@ -103,12 +103,20 @@ sub run() {
         last;
     }
 
+    # check for right boot-device
+    select_console 'install-shell';
```
Comment (okurz) [inline tests/installation/install_and_reboot.pm]: ?

**Comment by Soulofdestiny** [comment]
updated @okurz

File: tests/installation/install_and_reboot.pm [hunk 3]
```diff
@@ -103,13 +103,22 @@ sub run() {
         last;
     }
 
-    # Upload logs before reboot
+    # Stop reboot countdown for e.g. uploading logs
     if (!get_var("REMOTE_CONTROLLER")) {
         do {
             send_key 'alt-s';
         } until (wait_still_screen(2, 4));
+
```
Comment (okurz) [inline tests/installation/install_and_reboot.pm]: please no changing back and forth of unrelated whitespace. Add some rules to perltidy if you prefer this and convince everybody

**Comment by okurz** [comment]
please describe the "why" in your commit
